src/client/app/admin/user-management/user-signed-up.component.html

```diff
@@ -0,0 +1,13 @@
+<section>
+
+	<div class="row">
+		<h3 class="col-md-12 text-center">Account successfully requested</h3>
```

Comment:
I think the fact that we don't login on signup is just a bug. The issue with the starter is if we did fix it, it just dumps you on the "logged in" page. I think having a "Thanks for signing up" page is a good idea, but maybe we should have a discussion/sync about the whole "no account -> request account -> account approved -> logged in" flow should work.